lynx/common/enitity.py

```diff
@@ -12,4 +12,17 @@ class Exported(Serializable):
         args: str = ""
 
     def export(self) -> Exported:
```

Comment:
Just my way of cleaning up logic using helper functions. Still, export would only be ever called in `serialize`, so this may indded produce too much noise. I will remove it.